### ISSUE TITLE
Split article-level coding into enable and activate PRs

### ISSUE THREAD

**Issue by davidskalinder** [davidskalinder/mpeds-coder#82]
Enable batch should be almost done already (just need to change the comments for the article label handler in the controller)...

**Comment by davidskalinder** [comment]
https://github.com/MPEDS/mpeds-coder/pull/29
https://github.com/MPEDS/mpeds-coder/pull/30

**Comment by davidskalinder** [comment]
Hmm I just now noticed that these branches have not been merged into our master or testing branches.  I don't think there are any substantive changes -- mostly just changing template html for real html, and moving one function.  Still though, it should be merged with some care (maybe when new coders hit the testing deployment?).  I'm going to re-open this to get it merged properly.

**Comment by davidskalinder** [comment]
> Hmm I just now noticed that these branches have not been merged into our master or testing branches. I don't think there are any substantive changes -- mostly just changing template html for real html, and moving one function. Still though, it should be merged with some care (maybe when new coders hit the testing deployment?). I'm going to re-open this to get it merged properly.

Ah, and I know why: I was waiting for it to bubble up from upstream.  But that's waiting on DB migration; so I think I should go ahead and merge them in now but then keep the branches until the PRs get accepted.

**Comment by davidskalinder** [comment]
7e7172857 merges `enable_article_level` (via `change_article_gets_to_posts`) into `testing`; 61ffbbc1d3e merges `activate_article_level` into `testing`.

**Comment by davidskalinder** [comment]
No apparent problems with this after a week of testing with training articles.
